content/blog/2025-05-03-blog.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "TraceMonkey: A Tracing Just-In-Time Compiler for JavaScript"
+[extra]
+bio = """
+  Ethan Uppal Cornell CS '27
+  Tean Lai Cornell CS '26
```

Comment:
This is the place where you're supposed to include some brief information about yourself, maybe some links, etc. Like [this](https://www.cs.cornell.edu/courses/cs6120/2023fa/blog/lowering/):
<img width="705" alt="Screenshot 2025-05-05 at 4 43 10 PM" src="https://github.com/user-attachments/assets/80f4994c-b290-4766-8d3f-d3e46674e5f3" />